pkg/sql/colexec/insert/insert.go

```diff
@@ -96,7 +97,21 @@ func Call(_ int, proc *process.Process, arg any) (bool, error) {
 
 		}
 	}
-	err := n.TargetTable.Write(ctx, bat)
+	txnOperator, err := proc.TxnClient.New()
+	if err != nil {
+		return false, err
+	}
+	err = n.TargetTable.Write(ctx, bat)
+	if err != nil {
+		if err2 := txnOperator.Rollback(ctx); err2 != nil {
+			return false, err2
+		}
+		return false, err
+	}
+	err = txnOperator.Commit(ctx)
```

Comment:
How can I know this commit only happens for load?

Reply:
I fix it now